Bio/Align/stockholm.py

```diff
@@ -234,7 +234,12 @@ def _store_per_file_annotations(alignment, gf, rows):
             else:
                 assert len(value) == 1, (key, value)
                 value = value.pop()
-            alignment.annotations[AlignmentIterator.gf_mapping[key]] = value
+            if key in AlignmentIterator.gf_mapping:
+                alignment.annotations[AlignmentIterator.gf_mapping[key]] = value
+            else:
+                from warnings import warn
+
+                warn("Unknown GF annotation %s found, it will be ignored" % key)
```

Comment:
I am not a big fan of warnings, as by default they are issued only once.

Reply:
Warning once per key (with the default warning configuration) sounds a reasonable compromise to me. If you wanted even more warnings, including the problematic line would do that.

Reply:
> Warning once per key (with the default warning configuration) sounds a reasonable compromise to me. 

It's still confusing. When retrying the same example, the warning won't be issued and it may appear that the problem as disappeared.

In similar cases, we have added a `strict=True|False` argument to `parse` / `read` that controls whether an exception should be raised if an error occurs, or if errors should be silently ignored. We can apply the same logic here, but I would leave that for a separate PR.

Reply:
Yes, opening the same file within the same Python session will only show the warning once.

OK, then it is probably better to stay consistent. I will remove the warning.